src/hotspot/share/runtime/javaThread.hpp

```diff
@@ -1175,8 +1175,8 @@ class JavaThread: public Thread {
   bool is_interp_only_mode()                { return (_interp_only_mode != 0); }
   int get_interp_only_mode()                { return _interp_only_mode; }
   int set_interp_only_mode(int val)         { return _interp_only_mode = val; }
```

Comment:
Ther get_interp_only_mode() /set_interp_only_mode(int val)  also might be eliminated and replaced by set/clear instead.

Reply:
Good suggestion, thanks. Updated, it is being tested now.